ssh/ssh.go

```diff
@@ -238,6 +245,20 @@ func (ctx *SSHContext) ActivateConfiguration(host Host, configuration string, ac
 	return nil
 }
 
+func (ctx *SSHContext) HostCondor(host Host) error {
+
+	cmd, err := ctx.Cmd(host, "test", "-f", "/.morph_condor")
```

Comment:
I'm not sure how I feel about the hardcoded root of the filesystem path. First question: Should the path be configurable? Second: Can we find some other default-path which is more likely to always be writable by all users? Third: Should it even be possible to do this for any other than root, or `nix-trusted-users`? If not, we should make an explicit effort in requesting privileges here.